scripts/setup-annotations.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+
+# Determine PaSh top directory
+PASH_TOP=$(dirname "$(realpath "$0")")
+
+# Default annotations directory (sibling of PASH_TOP)
+DEFAULT_ANNOTATIONS_DIR="$(dirname "$PASH_TOP")/annotations"
+
+# Use user-provided path if given, otherwise use default
+if [ -n "$1" ]; then
+    ANNOTATIONS_DIR="$1"
+    ANNOTATIONS_DIR=$(realpath "$ANNOTATIONS_DIR")
+else
+    ANNOTATIONS_DIR="$DEFAULT_ANNOTATIONS_DIR"
+fi
+
+echo "Annotations directory set to: $ANNOTATIONS_DIR"
+
+# Always clone the repository to the specified or default location
+echo "Downloading annotations repository..."
+rm -rf "$ANNOTATIONS_DIR"
```

Comment:
I would not do that by default, I would either add a flag to remove (like --force or -f) or print an error if the directory already exists.